src/erlcloud.app.src

```diff
@@ -18,7 +18,9 @@
                   %% hackney,
                   lhttpc]},
   {modules, []},
-  {env, []},
+  {env, [
+      {services_vpc_endpoints, []}
```

Comment:
please add an example of one VPC endpoint entry.

Reply:
added ^